.clang-format

```diff
@@ -0,0 +1 @@
+BasedOnStyle: Google
```

Comment:
This should get us most of the way there (from https://github.com/itgmania/itgmania/pull/98)
```
BasedOnStyle: Google,
AlignAfterOpenBracket: AlwaysBreak,
DerivePointerAlignment: false,
PointerAlignment: Left,
QualifierAlignment: Left,
ReferenceAlignment: Left,
InsertBraces: true
```